tests/integration/test_iivsearch.py

```diff
@@ -137,23 +162,36 @@ def test_brute_force(tmp_path, model_count, start_modelres):
 
 @pytest.mark.filterwarnings("ignore::UserWarning")
 @pytest.mark.parametrize(
-    'iiv_strategy',
-    ['add_diagonal', 'fullblock'],
+    ('algorithm', 'correlation_algorithm', 'iiv_strategy', 'no_of_candidate_models'),
+    (
+        ('top_down_exhaustive', 'skip', 'add_diagonal', 15),
+        ('top_down_exhaustive', 'skip', 'fullblock', 15),
+        ('bottom_up_stepwise', 'skip', 'add_diagonal', 6),
+        ('bottom_up_stepwise', 'skip', 'fullblock', 6),
+    ),
 )
-def test_no_of_etas_iiv_strategies(tmp_path, model_count, start_modelres, iiv_strategy):
+def test_no_of_etas_iiv_strategies(
```

Comment:
Is it needed to add both the tests for bottom up? Was thinking since these are quite slow. Perhaps enough with `add_diagonal`?

Reply:
I can remove the `fullblock` for the bottom up approach but then that will not have coverage. Should both arguments still be tested for the top down approach?

Reply:
Kept both for now. Please let me know if you wish to have one removed

Reply:
I think when dry-run is implemented we can move them for the coverage. Do you know how long the added tests take currently?